src/Modules/Reports/Containers/reports.jsx

```diff
@@ -7,6 +7,8 @@ import VerticalTab       from 'App/Components/Elements/VerticalTabs/vertical-tab
 import AppRoutes         from 'Constants/routes';
 import { connect }       from 'Stores/connect';
 import WalletInformation from './wallet-information.jsx';
+// eslint-disable-next-line import/no-unresolved
+import 'Sass/app/modules/reports.scss';
```

Comment:
Does the `Sass` folder name need to be in uppercase?

Reply:
Yeah, I have had problems with the casings of the directory names, I'm double checking everything in the linux.